students/jstrauss123/lesson03/sequence_slicing.py

```diff
@@ -2,93 +2,32 @@
 
 # function - accept a sequence and return a copy of sequence with first and last items exchanged
 def exchange_first_last(seq):
-    # assign first and last value
-    first = seq[0]
-    last = seq[-1]
-    # create a working list
-    work_list = list(seq)
-    # assign first to last and last to first value
-    work_list[0] = last
-    work_list[-1] = first
-    # test seq type if str convert back to str
-    if type(seq) == str:
-        # convert list back to string
-        outseq = str("".join(work_list))
-    elif type(seq) == tuple:
-        outseq = tuple(work_list)
-    else:
-        outseq = work_list
+    outseq = seq[-1::] + seq[1:-1:] + seq[0:1:]
     return(outseq)
     
+    
 # function - accept a sequence and return every other item in sequence
 def  remove_every_other_item(seq):
-    work_list = list(seq)
-    work_list = work_list[0:-1:2]
-    if type(seq) == str:
-        # convert list back to string
-        outseq = str("".join(work_list))
-    elif type(seq) == tuple:
-        outseq = tuple(work_list)
-    else:
-        outseq = work_list
+    outseq = seq[0:-1:2]
     return(outseq)
     
-    
-# function - accept a sequence and maniuplate first and last four, return what's left every other
+# function - accept a sequence and manipulate first and last four, return what's left every other
 def  first_last_four(seq):    
-    work_list = list(seq)
-    # remove first and last four items from list
-    work_list = work_list[4:-4]
-    # remove every other from list
-    work_list = work_list[0:-1:2]
-    if type(seq) == str:
-        # convert list back to string
-        outseq = str("".join(work_list))
-    elif type(seq) == tuple:
-        outseq = tuple(work_list)
-    else:
-        outseq = work_list
+    outseq = seq[4:-4]
+    outseq = outseq[0:-1:2]
```

Comment:
this can be simplified to `return seq[4:-4:2]` , also no need for parens with `return` since it's a statement.